src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -228,25 +229,28 @@
         :disabled?           processing
         :accessibility-label :show-profiles}
        :i/multi-profile]]
-     [rn/scroll-view
+     [(if keycard-pairing rn/view rn/scroll-view)
       {:keyboard-should-persist-taps :always
        :style                        {:flex 1}}
       [quo/profile-card
        {:name                name
         :customization-color (or customization-color :primary)
         :profile-picture     profile-picture
         :card-style          style/login-profile-card}]
-      [password-input]]
-     [quo/button
-      {:size                40
-       :type                :primary
-       :customization-color (or customization-color :primary)
-       :accessibility-label :login-button
-       :icon-left           :i/unlocked
-       :disabled?           (or (not sign-in-enabled?) processing)
-       :on-press            login-multiaccount
-       :container-style     {:margin-bottom (+ (safe-area/get-bottom) 12)}}
-      (i18n/label :t/log-in)]]))
+      (if keycard-pairing
+        [keycard.pin/login]
+        [password-input])]
+     (when-not keycard-pairing
+       [quo/button
+        {:size                40
+         :type                :primary
+         :customization-color (or customization-color :primary)
+         :accessibility-label :login-button
+         :icon-left           :i/unlocked
+         :disabled?           (or (not sign-in-enabled?) processing)
+         :on-press            login-multiaccount
+         :container-style     {:margin-bottom (+ (safe-area/get-bottom) 12)}}
```

Comment:
idk it's not my code

Reply:
Sure, but this is a common improvement to think about. Magic values are a very common source of bugs in this repo. It's just a matter of time until a dev changes one magic value, but forgets to change the other.

But it was just a question, if you don't know and it would take too much time to check it's fine by me 👍🏼